TombOfSargeras/Locales/ruRU.lua

```diff
@@ -55,42 +55,42 @@ if L then
 	L.custom_on_stop_timers = "Всегда показывать полосы для способностей"
 	L.custom_on_stop_timers_desc = "Аватара Падшего использует свои способности случайным образом. Когда эта опция включена, полосы для этих способностей останутся на вашем экране."
 
-	--L.energy_leak = "Energy Leak"
-	--L.energy_leak_desc = "Display a warning when energy has leaked onto the boss in stage 1."
-	--L.energy_leak_msg = "Energy Leak! (%d)"
+	L.energy_leak = "Утечка энергии"
+	L.energy_leak_desc = "Отобразить предупреждение, когда энергия попала в босса на 1-й фазе"
+	L.energy_leak_msg = "Утечка Энергии! (%d)"
 
-	--L.warmup_trigger = "The husk before you" -- The husk before you was once a vessel for the might of Sargeras. But this temple itself is our prize. The means by which we will reduce your world to cinders!
+	L.warmup_trigger = "Аватара перед тобой!" -- The husk before you was once a vessel for the might of Sargeras. But this temple itself is our prize. The means by which we will reduce your world to cinders!
```

Comment:
Locales with `trigger` in their name require the translation to match the in-game text. Are you sure that exclamation mark is there in the russian translation?
This translation is the beginning of the yell before Fallen Avatar is attackable.

Reply:
I'll fix it